app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -914,6 +914,20 @@ public void onTrimMemory(int level) {
         }
     }
 
+    private void quiteApp() {
```

Comment:
I still prefer what I suggested :smile: this is showing a dialog not exiting the app, that's one of the potential outcomes of clicking a button, but not what this method does. 

In any case if you don't like it, `quiteApp` does not look good (I understand you could not use `quitApp` because that's already used)

Reply:
It was a typo :) but fair enough. Lets go with ShowExitAppDialog